lib/model/model.go

```diff
@@ -1602,12 +1608,18 @@ func (m *Model) localChangeDetected(folderCfg config.FolderConfiguration, files
 		// for windows paths, strip unwanted chars from the front.
 		path := filepath.Join(path, filepath.FromSlash(file.Name))
 
-		events.Default.Log(events.LocalChangeDetected, map[string]string{
-			"folderID": folderCfg.ID,
-			"label":    folderCfg.Label,
-			"action":   action,
-			"type":     objType,
-			"path":     path,
+		typeOfEvent := events.RemoteChangeDetected
+		if localChange {
+			typeOfEvent = events.LocalChangeDetected
+		}
```

Comment:
Given this boolean is used only to decide the event type, I feel that `diskChangeDetected` should just accept a event type instead of the boolean. 
This way you don't need to specify what the boolean means as it's obvious from the event value being passed.